docs/src/test/scala/docs/http/scaladsl/UnmarshalSpec.scala

```diff
@@ -28,4 +28,21 @@ class UnmarshalSpec extends AkkaSpec {
     //#use-unmarshal
   }
 
+  "use unmarshal without execution context" in {
+    //#use-unmarshal
```

Comment:
Ah yeah I wasn't quite sure what those comments were used for. Do they mean that the test is added automatically to the documentation?

I would tend to think a full example is not necessary.
However, would like me to add a line in the Unmarshall documentation where we can suggest the executionContext  is optional? Or should the Scala doc be enough?